rpc/src/tests/ws.rs

```diff
@@ -44,6 +43,7 @@ pub fn serve() -> (Server<ws::Server>, usize, GuardedAuthCodes) {
 		extractors::WsExtractor::new(Some(&authcodes.path)),
 		extractors::WsStats::new(stats),
 	).unwrap());
+	let port = res.addr().port() as usize;
```

Comment:
From `jsonrpc-ws-server`, looks like this `addr` is directly what's parsed in `address` variable. Will that result in `port == 0`?

Reply:
Should be fine, we get `local_addr` after `bind()` in ws as well.
https://github.com/paritytech/jsonrpc/blob/2ed142d55662155b0dc76b9b5d2732300f265dd6/ws/src/server.rs#L85

Reply:
It returns the assigned port; addr is initialised in the returned struct with a call to local_addr()